quiz/web/src/router.tsx

```diff
@@ -1,13 +1,26 @@
+import { UserRequired } from "modules/common/components";
 import { createBrowserRouter } from "react-router-dom";
 
 // eslint-disable-next-line import/prefer-default-export
 export const router = createBrowserRouter([
     {
         path: "/",
-        element: <h1>Home</h1>,
+        element: (
```

Comment:
no `/game/` endpoint ? 🥺

Reply:
I've assumed home to be the game endpoint. If you start an app http://azeno.com/ you should go to select game mode (home).

Reply:
if you don't have an account `<UserRequired>` will redirect you to start.